### PR TITLE
AGMS-537 - Review all images prior to GA

### PR DESCRIPTION
**Pull request by jstaffor**
## Motivation
https://issues.jboss.org/browse/AGMS-282

## What
https://issues.jboss.org/browse/AGMS-282

## Why
https://issues.jboss.org/browse/AGMS-282

## How
https://issues.jboss.org/browse/AGMS-282

### CODE REVIEW

File: modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc [hunk 1]
```diff
@@ -33,10 +33,7 @@ image::mobile-clients-service-bind-3.png[]
 NOTE: You can safely close the dialog, the process continues in the background.
 
 After the binding is complete, the configuration in `mobile-services.json` for the {mobile-client} is updated in the
-*Configuration* tab.
-This configuration is used by your app to interact with the bound services.
-
-image::mobile-clients-overview-mobile-services-json.png[]
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]: Hi @finp , following on from our discussion at the stand-up this morning, I reviewed the text that relates to the image and I think that the image isn't adding value and as such, I've removed it. If you think it's best to remove this image too, I'll go ahead and delete the actual png file and merge. Can you let me know what you think?

Comment (finp) [reply modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]: @jstaffor  go ahead and delete image.

File: modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc [hunk 2]
```diff
@@ -33,10 +33,7 @@ image::mobile-clients-service-bind-3.png[]
 NOTE: You can safely close the dialog, the process continues in the background.
 
 After the binding is complete, the configuration in `mobile-services.json` for the {mobile-client} is updated in the
```
Comment (finp) [inline modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]:
```suggestion
After the binding is complete, the JSON in the *Configuration* tab is automatically updated.
```

File: modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc [hunk 3]
```diff
@@ -33,10 +33,7 @@ image::mobile-clients-service-bind-3.png[]
...
-*Configuration* tab.
-This configuration is used by your app to interact with the bound services.
-
-image::mobile-clients-overview-mobile-services-json.png[]
+*Configuration* tab, and this configuration is used by your app to interact with the bound services.
```
Comment (finp) [inline modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]:
```suggestion
Putting this JSON into the `mobile-services.json` file for your mobile app enables your app to communicate with bound mobile services.
```

File: modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc (hunk 1 above)
Comment (jstaffor) [reply modules/ROOT/pages/_partials/mobile-client-using-mobile-services.adoc]: :+1:

**Comment by jstaffor** [comment]
Both https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/assets/images/send1.png and https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/assets/images/send2.png have been cleared as ok[1] by Eng. 

[1]
https://chat.google.com/room/AAAA_u92fRU - So it is fine as is, but as a practice I have no problem with it being obfuscated.

If you are happy with the images staying as is, can you approve this PR @finp ? If not, let me know please.

**Comment by jstaffor** [comment]
Cheers @finp :+1:

**Pull request merged**